compiler/src/iree/compiler/Pipelines/Options.cpp

```diff
@@ -119,17 +136,28 @@ void PreprocessingOptions::bindOptions(OptionsBinder &binder) {
       llvm::cl::cat(category));
 }
 
+void GlobalOptimizationOptions::applyOptimization(
+    const OptionsBinder &binder, const GlobalPipelineOptions &globalLevel) {};
```

Comment:
```suggestion
    const OptionsBinder &binder, const GlobalPipelineOptions &globalLevel) {}
```

this method shouldn't be needed anymore it seems?

Reply:
nope, I will remove it. This also wouldn't work with plugins